articles/ml_system_design/real_time_news_search_using_upstash_kafka_and_vector/Makefile

```diff
@@ -0,0 +1,49 @@
+RED := \033[0;31m
+BLUE := \033[0;34m
+GREEN := \033[0;32m
+YELLOW := \033[0;33m
+RESET := \033[0m
+
+export PYTHON_KEYRING_BACKEND=keyring.backends.fail.Keyring
+
```

Comment:
maybe inject this env var only in the CLI commands that require it?

Reply:
Will inject it before poetry install